src/weas_widget/weas.py

```diff
@@ -0,0 +1,89 @@
+from .base_widget import BaseWidget
+from .utils import ASE_Adapter, Pymatgen_Adapter, load_online_example
+from .atoms_viewer import AtomsViewer
+from .camera import Camera
+from .plugins.instanced_mesh_pritimive import InstancedMeshPrimitive
+import time
+import threading
+
+
+class WeasWidget:
+    def __init__(self, **kwargs):
+        self._widget = BaseWidget(**kwargs)
+        self.avr = AtomsViewer(self._widget)
+        self.camera = Camera(self._widget)
+        self.imp = InstancedMeshPrimitive(self._widget)
+
+    def _repr_mimebundle_(self, *args, **kwargs):
+        # if ipywdigets > 8.0.0, use _repr_mimebundle_ instead of _ipython_display_
+        if hasattr(self._widget, "_repr_mimebundle_"):
+            return self._widget._repr_mimebundle_(*args, **kwargs)
+        else:
+            return self._widget._ipython_display_(*args, **kwargs)
+
+    def from_ase(self, atoms):
+        self.avr.atoms = ASE_Adapter.to_weas(atoms)
```

Comment:
snake case usually does not extend to class names, I guess the conventional name would be `ASEAdapter`?

Reply:
Thanks for pointing out this. I changed them to `ASEAdapter` and `PymatgenAdapter`.